templates/spec.hbs

```diff
@@ -1,3 +1,4 @@
+%define debug_package %{nil}
```

Comment:
This was added as in centos/rocky8 it throw an error when looking for files which do not exist.
https://forums.centos.org/viewtopic.php?t=74540